### ISSUE TITLE
No Sound on Demos

### ISSUE THREAD

**Issue by djbase78** [Sakura-IT/ReWarp#10]
I tried ReWarp on my Sam440ep with PatchWarp3D and the Demos Sulaco and ByNight from Encore. Both running great but I have no Sound?

**Comment by rkujawa** [comment]
Maybe this is the same issue as #9, please take a look at #9 and see if the workaround from there works.

**Comment by djbase78** [comment]
No already tried this, pressing or holding mouse buttons has no effect.

**Comment by rkujawa** [comment]
@DvdBoon Any ideas here?

**Comment by djbase78** [comment]
I also see that the system freezes of I set Unit 0: in AHI preds.

**Comment by DvdBoon** [comment]
Sometimes, AHI prefs shows 2 possibilities regarding your soundcard. I have noticed that chosing one will bring sound to for example glQuake but not demo's and the other way around.

**Comment by DvdBoon** [comment]
As a side note, patchwarp3d should not be necessary anymore. Refer to ReWarp3DPPC and its ENV options.

**Comment by djbase78** [comment]
Hm, I tried ReWarp3D but freezes the system so PatchWarp3D is better working. But will give it another try.

**Comment by DvdBoon** [comment]
You have to set the ENV stuff in ENVARC:

OK tested bynight. It uses the Music Unit. What have you chosen there as Mode ID/driver?

**Comment by djbase78** [comment]
Are the any docs for the env stuff?

In AHI I set to cs4281:HiFi Stereo++

**Comment by DvdBoon** [comment]
ReWarp3DPPC supports the following arguments (in the form of a file) in ENVARC:Warp3DPPC/

PatchDrawElements: Use Alain Thellier's patch for W3D_DrawElements
PatchDrawArray: Use Alain Thellier's patch for W3D_DrawArray
PatchDrawLine: Use Alain Thellier's patch for W3D_DrawLine (Needed for FreeSpace)
PatchAll: Use all of the above

**Comment by DvdBoon** [comment]
I personally have a SAM440EP Flex sitting here. I am missing the adapter with the connectors for the  cs4281 so I put an SB Live in it. In AHI on Music Unit (so not unit 0) I have chosen EMU10kx HiFi 16bit stereo++.

I cannot test with the  cs4281 at the moment as i cannot see any place to connect my music installation so I am guessing it is missing some bracket with a connector.

You sure the Music Unit is not muted?

**Comment by djbase78** [comment]
No, everything looks ok, test sound in AHI Prefs works too. :-/

**Comment by djbase78** [comment]
I will try to get a sound card, maybe its just a driver issue.

**Comment by djbase78** [comment]
Finally tested with a TerraTec Aureon 5.1 and the Sound works. So might be a bug in onboard driver?

**Comment by DvdBoon** [comment]
AHI also have problems with the old library. I'm not sure how to fix this at the moment. See also the wipe out sound issue
